pages/about.html

```diff
@@ -4,5 +4,32 @@
 layout: about
 home: false
 ---
-
-<h2>About Content Goes Here</h2>
\ No newline at end of file
+<div class="aboutPage">
+  <div class="section">
+    <div class="infoContainer">
+      <h1>Who is Code for America?</h1>
+      <div class="text">
```

Comment:
Let's wrap any paragraphs in a `<p>` tag instead
I also don't believe we have a `text` class. These paragraphs are missing the correct styles.

Reply:
replaced divs with p tags. There's a `text` class in the main.scss file with font-size, line-height, and padding styling. I was following the styling from the Case Study section in Figma.